apps/pipeline-visualization/src/mermaid_utils.ts

```diff
@@ -0,0 +1,156 @@
+import { 
+    JayveeModel, 
+    JayveeServices, 
+    createJayveeServices, 
+    PipelineDefinition, 
+    getBlocksInTopologicalSorting,
+    collectStartingBlocks,
+    collectChildren, 
+    BlockDefinition
+  } from '@jvalue/jayvee-language-server';
+import { integer } from 'vscode-languageserver-protocol';
+
+import { 
+  diagramType,
+  diagramDirection, 
+  subgraphDirection,
+  subgraphColor,
+  //showComposite,
+  //showProperties,
+  properties,
+  font,
+  fontSize
+  } from './mermaid_params';
+
+
+  export interface MermaidOptions {
+    composite: boolean;
```

Comment:
There is a mismatch between the interface that commander creates and the defined interface.

```suggestion
    compositeBlocks: boolean;
```